appservice/src/remoteDebug/startRemoteDebug.ts

```diff
@@ -83,44 +84,92 @@ async function startRemoteDebugInternal(context: IActionContext, site: ParsedSit
     });
 }
 
-const baseConfigs: { [key in RemoteDebugLanguage]: Object } = {
-    [RemoteDebugLanguage.Node]: {
-        type: 'node',
-        protocol: 'inspector'
-    },
-    [RemoteDebugLanguage.Python]: {
-        type: 'python'
-    }
-};
-
-async function getDebugConfiguration(context: IActionContext, language: RemoteDebugLanguage): Promise<vscode.DebugConfiguration> {
-    if (!(language in baseConfigs)) {
-        throw new Error(localize('remoteDebugLanguageNotSupported', 'The language "{0}" is not supported for remote debugging.', language));
+async function selectMultirootWorkspaceTarget(): Promise<vscode.Uri | undefined> {

```

Comment:
You don't have to implement this yourself, VS Code has an existing method `vscode.window.showWorkspaceFolderPick`

Reply:
Ah, fixed.